aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPlugin.java

```diff
@@ -72,9 +73,9 @@ public final class AWSPinpointAnalyticsPlugin extends AnalyticsPlugin<Object> {
     private static final String AUTH_DEPENDENCY_PLUGIN_KEY = "awsCognitoAuthPlugin";
 
     private final Application application;
-    private AutoEventSubmitter autoEventSubmitter;
+    private AutoEventSubmitterOld autoEventSubmitter;
```

Comment:
Instead of Old should we use "legacy" ?

Reply:
This file will be deleted in the following PR so probably wont change this.

Reply:
This file will be deleted in the next PR.